CONTRIBUTING.md

```diff
@@ -1,39 +1,167 @@
+its being close is just an opportunity to help out some more, and then re-open.
+
+#### After
+
+Thanks to everyone involved!
+
+If you get help, sharing it back in the form of a pull-request or making an issue to document
+what you've found is *extremely* helpful.
+
+If you solve your issue, stop working on it, or realize the problem was something else,
+please share that in a comment to an issue and close it.  That way, everyone can learn and
+we don't have closed issues without a clear resolution. Even if it's just a stackoverflow link :)
+And please don't forget to stay involved in the issue until it is closed! Thanks to all!
+
+### Writing code and comments
+
+- We are actively working to identify tasks under the label [**Good for New
+  Contributors**](https://github.com/rails-api/active_model_serializers/labels/Good%20for%20New%20Contributors).
+  Some bugs are expressly not good for new contributors, so don't expect 100% overlap between the two.
+  - [Changelog
+      Missing](https://github.com/rails-api/active_model_serializers/issues?q=label%3A%22Changelog+Missing%22+is%3Aclosed) is
+    an easy way to help out.
+
+- If you want to work on new feature development, look for the label [**Feature**](https://github.com/rails-api/active_model_serializers/labels/Feature).
+
+- We are also encouraging comments to substantial changes (larger than bugfixes and simple features) under an
+  "RFC" (Request for Comments) process before we start active development.
+   Look for the [**RFC**](https://github.com/rails-api/active_model_serializers/labels/RFC) label.
 
 ## Issue Labeling
 
-AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIssueLabels) for Github Issues. You can [see our labels here](https://github.com/rails-api/active_model_serializers/labels).
-
-## Contributing
-
-1. Fork it ( https://github.com/rails-api/active_model_serializers/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Write tests for your feature, or regression tests highlighting a bug
-4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
-8. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
-with a brief description of any breaking changes, fixes, features, or
-miscellaneous changes under the proper version section.
-9. Iterate on feedback given by the community (fix syntax, modify bits of code, add
+ActiveModelSerializers uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIssueLabels) for Github Issues. You can [see our labels here](https://github.com/rails-api/active_model_serializers/labels).
+
+## Submitting a pull request (PR)
+
+1. The vast majority of development is happening under the `master` branch, currently slated for release as `0.10.x`.
```

Comment:
should the version mention here be removed? so that this doc doesn't *need* to be updated with each release?